build/ci/linux/build.sh

```diff
@@ -39,11 +40,16 @@ while [[ "$#" -gt 0 ]]; do
         --build_mode) BUILD_MODE="$2"; shift ;;
         --youtube_api_key) YOUTUBE_API_KEY="$2"; shift ;;
         --arch) PACKARCH="$2"; shift ;;
+        --qt5_compat) QT5_COMPAT="$2"; shift ;; 
         *) echo "Unknown parameter passed: $1"; exit 1 ;;
     esac
     shift
 done
 
+if [ "$QT5_COMPAT" != "ON" ]; then 
+    echo "for 4.3 branch only Qt5 (QT5_COMPAT == 'ON')"
```

Comment:
Can we hardcode qt5_compat for 4.3 instead of showing warning and continuing with qt6?

Reply:
it is hardcoded in the cmake project, and here it is not pass to next